.cascade-code/Chapman.edu/_cascade/formats/modular/mastheads/side_panel_hero.vm

```diff
@@ -6,124 +6,127 @@
+    <div class="cd-hero">
+      <div class="cd-arrow-left"><a href="#">&#139;</a></div>
+      <div class="cd-arrow-right"><a href="#">&#155;</a></div>
+      <div class="cd-slide-container">
+        <ul class="cd-hero-slider autoplay">
+          #foreach($slide in $slides)
+            ## Background Variables
+            #set ( $backgroundType          = $slide.getChild('featuredBackgroundMedia').getChild('type').value )
+            #set ( $backgroundImage         = $slide.getChild('featuredBackgroundMedia').getChild('imageFile').getChild('path').value )
+            #set ( $backgroundImageAlignY   = $slide.getChild('featuredBackgroundMedia').getChild('alignY').value )
+            #set ( $backgroundVideo         = $slide.getChild('featuredBackgroundMedia').getChild('videoFile').getChild('path').value )
+            #set ( $backgroundFallbackImage = $slide.getChild('featuredBackgroundMedia').getChild('fallbackImageFile').getChild('path').value )
+            #set ( $backgroundAltText       = $slide.getChild('featuredBackgroundMedia').getChild('alternateText').value )
 
-                ## Featured Content Variables
-                #set ( $featuredCategory      = $slide.getChild('featuredPanel').getChild('category').value )
-                #set ( $featuredTitle         = $slide.getChild('featuredPanel').getChild('title').value )
-                #set ( $featuredSubtitle      = $slide.getChild('featuredPanel').getChild('subtitle').value )
-                #set ( $featuredBody          = $_SerializerTool.serialize($slide.getChild('featuredPanel').getChild('body'), true) )
-                #set ( $featuredLink          = $slide.getChild('featuredPanel').getChild('link') )
-                #set ( $featuredContentExists = "#if($featuredCategory != '' || $featuredTitle != '' || $featuredSubtitle != '' || $slide.getChild('featuredPanel').getChild('body').value.length() != 0 || $featuredLink.getChild('label').value != '')true#{else}false#end")
+            ## Featured Content Variables
+            #set ( $featuredCategory      = $slide.getChild('featuredPanel').getChild('category').value )
+            #set ( $featuredTitle         = $slide.getChild('featuredPanel').getChild('title').value )
+            #set ( $featuredSubtitle      = $slide.getChild('featuredPanel').getChild('subtitle').value )
+            #set ( $featuredBody          = $_SerializerTool.serialize($slide.getChild('featuredPanel').getChild('body'), true) )
+            #set ( $featuredLink          = $slide.getChild('featuredPanel').getChild('link') )
+            #set ( $featuredContentExists = "#if($featuredCategory != '' || $featuredTitle != '' || $featuredSubtitle != '' || $slide.getChild('featuredPanel').getChild('body').value.length() != 0 || $featuredLink.getChild('label').value != '')true#{else}false#end")
+            #set ( $featuredLinkExists    = $featuredLink.getChild('label').value != '' )
```

Comment:
The majority of the stuff here is just indentation that I fixed. The only things I actually changed were this line (new variable called `$featuredLinkExists`) and lines 83 and 102 where I used it to test whether link data definitions were left empty by the user. For accessibility, links cannot be empty.